src/frontend/web_application/src/scenes/Signin/components/SigninForm/presenter.jsx

```diff
@@ -37,10 +37,25 @@ class SigninForm extends Component {
     this.setState(generateStateFromProps(this.props));
   }
 
+  componentDidMount() {
+    setTimeout(this.setValues(), 1);
```

Comment:
is timeout useful regarding to the blog post you pointed to ?

Reply:
that's because setting state in `componentDidMount` is not possible without `setTimeout()`